docs/web-console-guide/master.adoc

```diff
@@ -112,6 +112,9 @@ include::topics/web-add-user.adoc[leveloffset=+2]
 
 // TODO: eventually add steps for reverting back to no auth required
```

Comment:
Is this TODO still relevant or needs to be wiped off?

Reply:
It seems to already be deleted.
@RichardHoch  is that correct?